crates/optimism/node/src/txpool.rs

```diff
@@ -63,20 +63,14 @@ impl OpPooledTransaction {
     pub fn estimated_compressed_size(&self) -> u64 {
         *self
             .estimated_tx_compressed_size
-            .get_or_init(|| tx_estimated_size_fjord(&self.inner.transaction().encoded_2718()))
+            .get_or_init(|| {
+                maili_protocol::tx_estimated_size_fjord(&self.inner.transaction().encoded_2718())
+            })
+            .try_into()
+            .unwrap_or_else(|_| u64::MAX)
```

Comment:
@refcell this should be changed to u64 in maili

Reply:
Fixing here: https://github.com/op-rs/maili/pull/115

If this is blocking, I can cut another quick release and fix up this pr.